symengine/lib/symengine_wrapper.pyx

```diff
@@ -2977,11 +3029,16 @@ zoo = c2py(symengine.ComplexInf)
 nan = c2py(symengine.Nan)
 true = c2py(symengine.boolTrue)
 false = c2py(symengine.boolFalse)
+try:
+    import sympy
```

Comment:
If `hbar` as an object under class `Constant` is defined in `SymEngine`, I'd follow the same routine that we follow for all the other objects. That's why I asked for implementing the same in the `Gitter` channel.